File: jgp-database-app/src/main/java/com/jgp/dashboard/dto/CountySummaryDto.java
package com.jgp.dashboard.dto;

import java.math.BigDecimal;
import java.time.LocalDate;

public record CountySummaryDto(
        String countyCode,
        String countyName,
        Integer businessesTrained,
        Integer businessesLoaned,
        BigDecimal amountDisbursed,
        BigDecimal outStandingAmount,
        LocalDate dataDate
) {
}

File: jgp-database-web/src/app/pages/dashboard/dto/county-summary-dto.ts
export interface CountySummaryDto {
    countyCode: string | undefined;
    countyName: string | undefined;
    approximateCenterLatitude: number | undefined;
    approximateCenterLongitude: number | undefined;
    businessesTrained: number | undefined;
    businessesLoaned: number | undefined;
    amountDisbursed: number | undefined;
    businessesMentored: number | undefined;
}